<ui version="4.0" >
 <class>PimGen</class>
 <widget class="QWidget" name="PimGen" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PIM Data generator</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Contacts</string>
     </property>
     <property name="buddy" >
      <cstring>contactCount</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSpinBox" name="contactCount" >
     <property name="maximum" >
      <number>20000</number>
     </property>
     <property name="singleStep" >
      <number>100</number>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Appointments</string>
     </property>
     <property name="buddy" >
      <cstring>appointmentCount</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QSpinBox" name="appointmentCount" >
     <property name="maximum" >
      <number>20000</number>
     </property>
     <property name="singleStep" >
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Tasks</string>
     </property>
     <property name="buddy" >
      <cstring>taskCount</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QSpinBox" name="taskCount" >
     <property name="maximum" >
      <number>20000</number>
     </property>
     <property name="singleStep" >
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QLabel" name="progressLabel" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>24</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>107</width>
       <height>171</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
